ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13.yaml

```diff
@@ -26,7 +26,7 @@ tests:
   steps:
     cluster_profile: gcp-qe
     env:
-      WORKER_REPLICA_COUNT: "20"
+      WORKER_REPLICA_COUNT: "24"
```

Comment:
think we need to do the same thing you are doing above, taking out the workers-scale chain and replace with create-infra-move-ingress-monitoring-registry

Reply:
Was thinking of the aws installation type. This is fine as is

Reply:
As discussed on Slack, the ipi/gcp code path doesn't allow us to modify the cluster size out of the gate... We can do a follow up patchset to update gcp ipi.